src/main/java/com/dao/impl/mysql/user/MySqlUserDAO.java

```diff
@@ -0,0 +1,151 @@
+                return user;
+            } catch (SQLException e) {
+                LOGGER.error("Unable to get connection: {" + e.getMessage() + "}", e);
+                throw new DAOException("Unable to get user");
+            }
+        }
+        return user;
+    }
+
+    @Override
+    public UserEntity create(UserEntity entity) throws DAOException {
+        Optional<Connection> connectionOptional = getConnection();
+        if (connectionOptional.isPresent()) {
+            try (Connection connection = connectionOptional.get();
+                 PreparedStatement statement = connection.prepareStatement(UserQuery.INSERT_USER)){
+                statement.setString(1, entity.getName());
+                statement.setString(2, entity.getSurname());
+                statement.setString(3, entity.getLogin());
+                statement.setString(4, entity.getPassword());
+                statement.setLong(5, 2);
+                statement.execute();
+                return getByLogin(entity.getLogin());
+            } catch (SQLException e) {
+                LOGGER.error("Unable to get connection: {" + e.getMessage() + "}", e);
+                throw new DAOException("Unable to create user");
+            }
+        }
+        return entity;
+    }
+
+    @Override
+    public UserEntity getEntity(long id) throws DAOException {
+        Optional<Connection> connectionOptional = getConnection();
+        UserEntity user = new UserEntity();
+        if (connectionOptional.isPresent()) {
+            try (Connection connection = connectionOptional.get();
+                 PreparedStatement statement = connection.prepareStatement(UserQuery.GET_USER_BY_ID)) {
+                statement.setLong(1, id);
+                ResultSet resultSet = statement.executeQuery();
+                while (resultSet.next()){
+                    user = getUserEntityFromResultSet(resultSet);
+                }
+                return user;
+            } catch (SQLException e) {
+                LOGGER.error("Unable to get connection: {" + e.getMessage() + "}", e);
+                throw new DAOException("Unable to get user");
+            }
+        }
+        return user;
+    }
+
+    @Override
+    public List<UserEntity> getAll() throws DAOException {
+        Optional<Connection> connectionOptional = getConnection();
+        if (connectionOptional.isPresent()) {
+            try (Connection connection = connectionOptional.get();
+                 Statement statement = connection.createStatement()) {
+                ResultSet resultSet = statement.executeQuery(UserQuery.GET_ALL_USERS_QUERY);
+                List<UserEntity> entityList = new ArrayList<>();
+                while (resultSet.next()){
+                    entityList.add(getUserEntityFromResultSet(resultSet));
+                }
+                return entityList;
+            } catch (SQLException e) {
+                LOGGER.error("Unable to get connection: {" + e.getMessage() + "}", e);
```

Comment:
Don't use string concatenation in logger expression, should be like this:

_LOGGER.error("Unable to get connection: {}", e.getMessage(), e);_

This "{}" value will be replaced by parameter, passed after message to the log method (e.getMessage()), log4j will handle it with better performance